examples/atmega328p/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "mega328-test"
+version = "0.1.0"
+authors = ["Frank Villaro-Dixon <frank@villaro-dixon.eu>"]
+edition = "2021"
+license = "Public domain"
+
+[[bin]]
+name = "mega328-test"
+test = false
+bench = false
+
+
+[dependencies]
+ufmt = "0.2.0"
+nb = "0.1.2"
+embedded-hal = "0.2.3"
+
+
+[dependencies.avr-device]
+version = "0.5.3"
```

Comment:
Sorry, didn't catch this the first time around:

Including `avr-device` from crates.io is a bit unusual as you'd usually want the examples of a library crate to use the crate that's part of the same repository.  This is very useful if you want to try an example against changes in the library code.

You could do this by replacing the `version = ` with a `path = "../.."` line.

**However**, for `avr-device` in particular, this isn't quite that simple:  The problem is that to use the repository version of `avr-device`, you first have to run all the `make` steps to generate the crate's code.  I wouldn't want that to be required for just running an example...

So my suggestion is this:  Let's keep the `version = ` line but add the `path = ` line next to it, in a comment.

```suggestion
version = "0.5.3"
# To use the local version of avr-device instead, uncomment the following line:
path = "../.."
```